NorthstarDLL/mods/autodownload/moddownloader.cpp

```diff
@@ -132,9 +132,13 @@ void FetchModSync(std::promise<std::optional<fs::path>>&& p, std::string_view ur
 	CURL* easyhandle;
 	easyhandle = curl_easy_init();
 
-	curl_easy_setopt(easyhandle, CURLOPT_TIMEOUT, 30L);
 	curl_easy_setopt(easyhandle, CURLOPT_URL, url.data());
 	curl_easy_setopt(easyhandle, CURLOPT_FAILONERROR, 1L);
+
+	/* abort if slower than 30 bytes/sec during 10 seconds */
```

Comment:
We only use `//` for single line comments AFAIK

```suggestion
	// abort if slower than 30 bytes/sec during 10 seconds
```